ideascube/mediacenter/management/commands/clean.py

```diff
@@ -32,6 +38,16 @@ def handle(self, *args, **options):
 
         options['func'](options)
 
+    def clean_media(self, options):
+        Document.objects.filter(package_id='').delete()
+        left_media_count = Document.objects.all().count()
+        if left_media_count:
+            print("{} media have been installed by packages."
+                  " They have been not deleted.\n"
+                  "You must delete the corresponding package if you want to "
+                  "remove them.".format(left_media_count))
```

Comment:
Maybe print the command? `'%s catalog remove pkgid [pkgid ...]' % args[0]` (not sure how to get the program name)